pymatgen/io/vasp/inputs.py

```diff
@@ -1828,7 +1848,7 @@ def from_file(cls, filename: str) -> PotcarSingle:
         Returns:
             PotcarSingle
         """
-        match = re.search(r"(?<=POTCAR\.)(.*)(?=.gz)", str(filename))
+        match = re.search(r"(?<=POTCAR\.)(.*)(?=.gz)", filename)
```

Comment:
i assume the `str(filename)` was there to handle `pathlib.Path` so let's keep it

Reply:
Thanks for reviewing @janosh. Sure let's keep the `str()` for now. I would resolve these conflicts and @esoteric-ephemera should have already reviewed this as the INCAR tag checking changes have been reverted, and would be pushed in a separate PR.